pkgs/development/python-modules/recipe-scrapers/default.nix

```diff
@@ -64,7 +64,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "Python package for scraping recipes data";
     homepage = "https://github.com/hhursev/recipe-scrapers";
-    changelog = "https://github.com/hhursev/recipe-scrapers/releases/tag/${version}";
+    changelog = "https://github.com/hhursev/recipe-scrapers/releases/tag/${src.tag}";
```

Comment:
That seems weird, given #367739.

Reply:
There's been no changes to nixpkgs-update which would cause this https://github.com/nix-community/nixpkgs-update/commits/main
https://github.com/search?q=repo%3Anix-community%2Fnixpkgs-update%20%2Fsrc.tag%2F&type=code

`python3Packages.recipe-scrapers.passthru.updateScript` did this https://github.com/NixOS/nixpkgs/blob/794d83cb94e9e9e7ac8d340829d379ce4a472fce/pkgs/by-name/up/update-python-libraries/update-python-libraries.py#L503

Reply:
Also it's fine to do that since finalAttrs isn't used but someone could change it to use finalAttrs without knowing it causes issues